### PR TITLE
feat: MOB-357 Show default tips if no data

### PR DESCRIPTION
**Pull request by seazon**

### CODE REVIEW

File: src/components/FPS/index.js [hunk 1]
```diff
@@ -170,12 +170,12 @@ const FPS = () => {
         {/* 2.3.2 chart */}
         <Text style={styles.subTitle}>2.3.2 Data Chart</Text>
         <Text style={styles.text}>A dot line chart is a common choice for FPS data because it can show changes over time or frames.</Text>
-        <View style={styles.chartDesBox} break>
+        <View style={styles.chartDesBox}>
           <View style={styles.chartContainer}><Image src={fpsImage} break /></View>
           <View style={styles.chartDesContainer}><Text style={styles.hint}>{chartDes}</Text></View>
         </View>
```
Comment (Tdreamy) [inline src/components/FPS/index.js:176]: remove 'break' maybe can cause chart and its des across two pages

File: src/components/MemoryLeak/index.jsx [hunk 2]
```diff
@@ -7,27 +7,21 @@ import grateUtil from 'src/utils/grade';
...
   const performanceData = useContext(Context);
   if (!performanceData) {
-    return null;
+    // return null;
   }
   const { memoryLeakData } = performanceData;
   if (!memoryLeakData || memoryLeakData.length < 1) {
-    return null;
+    // return null;
   }
```
Comment (Tdreamy) [inline src/components/MemoryLeak/index.jsx:20]: when no data, still have to render this compoent, these code can be removed

File: src/utils/grade.js [hunk 3]
```diff
@@ -48,6 +48,34 @@ const getNetworkMark = (sucsessRate, slowReqRate) => {
   return Math.round(sucsessRate * 50 + (1 - slowReqRate) * 50)
 }
 
+const getSlowRequestRate = (slowRequestCount, summaryRequestCount) => {
+  let slowReqRate
+  if (summaryRequestCount == 0) {
```
Comment (Tdreamy) [inline src/utils/grade.js:53]: you can use '===',  use '==' will build failed

File: src/components/FPS/index.js (hunk 1 above)
Comment (seazon) [reply src/components/FPS/index.js:176]: use `wrap={false}` to implement, `break` always start from new page.

File: src/components/MemoryLeak/index.jsx (hunk 2 above)
Comment (seazon) [reply src/components/MemoryLeak/index.jsx]: removed

**Review by Tdreamy — APPROVED** [review]

**Pull request merged**